crates/partition-store/src/tests/invocation_status_table_test/mod.rs

```diff
@@ -155,7 +140,7 @@ async fn populate_data<T: InvocationStatusTable>(txn: &mut T) {
 
     txn.put_invocation_status(
         &INVOCATION_ID_4,
-        &killed_status(INVOCATION_TARGET_4.clone()),
+        &invoked_status(INVOCATION_TARGET_4.clone()),
     )
     .await
```

Comment:
What's the purpose of another `invoked` invocation entry? Doesn't `INVOCATION_ID_1` is enough?